airbyte-cdk/java/airbyte-cdk/db-destinations/src/testFixtures/kotlin/io/airbyte/cdk/integrations/standardtest/destination/DestinationAcceptanceTest.kt

```diff
@@ -79,7 +79,7 @@ import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 abstract class DestinationAcceptanceTest {
-    protected lateinit var TEST_SCHEMAS: HashSet<String>
+    protected var TEST_SCHEMAS: HashSet<String> = HashSet()
```

Comment:
WTF is this mutable state... I had a deeper look and I didn't like what I saw. Out of scope of this PR obviously but please consider filing an issue in the destinations board to investigate and fix this, `DestinationAcceptanceTest` is an important test fixture that deserves to be improved.